configure.ac

```diff
@@ -293,10 +293,10 @@ int128_struct)
   AC_DEFINE(USE_FORCE_WIDEMUL_INT128_STRUCT, 1, [Define this symbol to force the use of the structure for simulating (unsigned) int128 based wide multiplication])
```

Comment:
This newly added `AC_DEFINE` should be changed, too.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/secp256k1/pull/1178#issuecomment-1353039879).